### PR TITLE
[UIK-2975][d3-chart, docs] Add accessible name to chart

### PR DESCRIPTION
**Pull request by ksimanov**
## Motivation and Context

A11y improvement of d3-chart component
## How has this been tested?

manually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Nice improve.

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly or it's not required.
- [ ] Unit tests are not broken.
- [x] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: semcore/d3-chart/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [3.58.5] - 2024-12-20
+
+### Added
+
+- `aria-hidden="true"` to `PatternSymbol` inside `LegendItem` to improve a11y.
```
Comment (sheila-semrush) [inline semcore/d3-chart/CHANGELOG.md:9]: I think we should listen to the linter and write "A11y" capitalized :)

**Review by sheila-semrush — APPROVED** [review]
added role='img' and aria-labels to the low level pattern component example

@Valeria-Zimnitskaya NVDA reads the first aria-label twice (at least in Firefox) in the low level pattern example, I think we can't do anything with it

File: website/docs/data-display/d3-chart/examples/low-level-components-use.tsx [hunk 2]
```diff
@@ -7,10 +7,12 @@ const Demo = () => {
   const patternSymbolSize = getPatternSymbolSize({ patternKey, patterns });
 
   return (
-    <svg height='100px' width='200px'>
+    <svg height='100px' width='200px' aria-label='PatternFill and PatternSymbol' role='img'>
       <PatternFill id='pattern-element' patternKey={patternKey} color='red' patterns={patterns} />
       <rect width='100px' height='100px' x='0' y='0' fill='url(#pattern-element)' stroke='red' />
       <PatternSymbol
+        aria-label='zigzag PatternSymbol'
+        role='img'
```
Comment (ilyabrower) [inline website/docs/data-display/d3-chart/examples/low-level-components-use.tsx:15]: is it ok to use `img` in `img`?

Comment (sheila-semrush) [reply website/docs/data-display/d3-chart/examples/low-level-components-use.tsx:15]: good question, but it seems that it's ok, Axe doesn't complain, and even MDN has nested images in examples https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role
I guess we'll see the final answer in the next LA recheck

Comment (sheila-semrush) [reply website/docs/data-display/d3-chart/examples/low-level-components-use.tsx]: gave it a bit of thought and changed role to "none" for the nested svg
the standard says img's children are supposed to be presentational
and it fixed the double reading in NVDA

File: semcore/d3-chart/CHANGELOG.md (hunk 1 above)
Comment (ksimanov) [reply semcore/d3-chart/CHANGELOG.md]: capitalized :)

**Review by ilyabrower — APPROVED** [review]

**Review by Valeria-Zimnitskaya — APPROVED** [review]

**Pull request merged**